README.md

```diff
@@ -44,6 +44,8 @@ Next follow these instructions.
     CREATE DATABASE osp;
     \c osp;
     GRANT ALL PRIVILEGES ON DATABASE osp to osp;
+    GRANT ALL PRIVILEGES ON DATABASE test_osp to osp;
```

Comment:
As the last two commands are not required for setting up the project database. Can you add a comment like these two commands is required for running the test on your device? or update it in the `test` section of readme instead of writing here.

Reply:
You are right!

Reply:
Resolved